dotcom-rendering/src/components/ListenToArticle.importable.tsx

```diff
@@ -7,26 +7,55 @@ import { ListenToArticleButton } from './ListenToArticleButton';
 type Props = {
 	articleId: string;
 };
-export const ListenToArticle = ({ articleId }: Props) => {
-	const [showButton, setShowButton] = useState<boolean>(false);
 
-	const isBridgetCompatible = useIsBridgetCompatible('8.5.1');
+export const formatAudioDuration = (durationInSeconds: number): string => {
+	const hours = Math.floor(durationInSeconds / 3600);
+	const minutes = Math.floor((durationInSeconds % 3600) / 60);
+	const seconds = durationInSeconds % 60;
+
+	if (hours > 3) {
```

Comment:
I think this means anything **4 hours** or more is invalid, rather than 3 hours? In other words, 03:00:00-03:59:59 would still get displayed, but anything longer wouldn't?

If you want anything over 3 hours not to show, as mentioned in the description, I think this would need to be:

```suggestion
	if (hours > 2) {
```

or

```suggestion
	if (hours >= 3) {
```

?